### PR TITLE
New block

### PR DESCRIPTION
**Pull request by ValerieLeonidova**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -11,4 +12,22 @@ footer {
 }
 .section-pink {
     background: pink;
+    height: 100vh;
+    padding: 32px;
+}
+.section-pink h2 {
```
Comment (Vilka-Valeri) [inline css/style.css:18]: empty line

File: index.html [hunk 2]
```diff
@@ -1,77 +1,86 @@
...
+        eveniet dolorum ducimus soluta nesciunt, dicta voluptates magnam,
+        aliquid obcaecati enim voluptatem. Deleniti ea quae asperiores voluptate
+        et non.
+      </p>
+      <ul>
+        <li>Lorem ipsum dolor sit amet consectetur adipisicing elit</li>
+        <li>Iure quas odit eveniet dolorum ducimus soluta nesciunt</li>
+        <li>Deleniti ea quae asperiores voluptate et non</li>
+      </ul>
+      <img src="./image/cats.jpg" alt="">
```
Comment (Vilka-Valeri) [inline index.html:74]: no width, empty alt

File: css/style.css (hunk 1 above)
Comment (ValerieLeonidova) [reply css/style.css:18]: done

File: index.html (hunk 2 above)
Comment (ValerieLeonidova) [reply index.html]: done

**Review by Vilka-Valeri — APPROVED** [review]

**Pull request merged**